tests/transforms/test_qcut.py

```diff
@@ -3697,10 +3731,37 @@ def circuit(template_weights):
 
         spy = mocker.spy(qcut.cut_circuit, "expand_fn")
         res = qnode_cut(template_weights)
-        assert spy.call_count == 2
+        assert spy.call_count == 1
```

Comment:
Also here and in line 3763 of `test_expansion_mc_ttn` - we expand the expansion to be happening twice because we have a nested block.